src/main/java/uk/gov/hmcts/ecm/common/service/pdf/et3/ET3FormHeaderMapper.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.ecm.common.service.pdf.et3;
+
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+
+import java.time.LocalDate;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentMap;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.DATE_FORMAT_DD_MM_YYYY_DASH;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.DATE_FORMAT_YYYY_MM_DD_DASH;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.SUBMIT_ET3;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.TXT_PDF_HEADER_FIELD_CASE_NUMBER;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.TXT_PDF_HEADER_FIELD_DATE_RECEIVED;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.TXT_PDF_HEADER_FIELD_RTF;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.ET3FormConstants.TXT_PDF_HEADER_VALUE_ADDITIONAL_DOCUMENT_EXISTS;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.util.ET3FormUtil.formatDate;
+import static uk.gov.hmcts.ecm.common.service.pdf.et3.util.ET3FormUtil.putPdfTextField;
+
+public final class ET3FormHeaderMapper {
+
+    private ET3FormHeaderMapper() {
+        // Add a private constructor to hide the implicit public one.
+    }
+
+    /**
+     * Maps header values (case number, case received date, RFT) with PDF input fields.
+     * @param caseData case data that is received by case number
+     * @param respondentSumType respondent data selected by representative of respondent
+     * @param pdfFields print fields that is created in ET3FormMapper
+     */
+    public static void mapHeader(CaseData caseData,
+                                 RespondentSumType respondentSumType,
+                                 ConcurrentMap<String, Optional<String>> pdfFields,
+                                 String event) {
+        putPdfTextField(pdfFields, TXT_PDF_HEADER_FIELD_CASE_NUMBER, caseData.getEthosCaseReference());
+        if (SUBMIT_ET3.equals(event)) {
```

Comment:
This needs to be extended to include the new et3 submit event

Reply:
The old submitEt3 needed to remain as now if a rep submits an ET3, this logic won't be applied